src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -44,121 +61,319 @@ export function formatDate(date) {
+
+      const pluralize = {
+        allowed: (allowedIssues.length > 1) ? 'these issues' : 'this issue',
+        denied: (deniedIssues.length > 1) ? 'issues' : 'issue',
+      };
+
+      let allowedBlock = null;
+      let deniedBlock = null;
+
+      if (allowedIssues.length) {
+        allowedBlock = (
+          <div>
+            <h5 className="allowed-items">Allowed</h5>
+            <p>The judge overrules the original decision and finds {pluralize.allowed} in your favor</p>
+            <ul>{allowedIssues}</ul>
+          </div>
+        );
+      }
+      if (deniedIssues.length) {
+        deniedBlock = (
+          <div>
+            <h5 className="denied-items">Denied</h5>
+            <p>The judge upholds the original decision for the following {pluralize.denied}</p>
+            <ul>{deniedIssues}</ul>
+          </div>
+        );
+      }
+
+      contents.title = 'The Board made a decision on your appeal';
+      contents.description = (
+        <div>
+          <p>
+            The Board of Veterans’ Appeals made a decision on your appeal. Here is an overview of
+            the decision:
+          </p>
+          <div className="decision-items">
+            {allowedBlock}
+            {deniedBlock}
+          </div>
+          <div>
+            For issues that are allowed, you will receive compensation. For more information, please
+            contact your VSO or representative.
+          </div>
         </div>
+      );
+      break;
+    }
+    case STATUS_TYPES.fieldGrant:
+      contents.title = 'The Veterans Benefits Administration granted your appeal';
+      contents.description = (
+        <p>The Veterans Benefits Administration agreed with you and decided to overturn the
+        original decision. If this decision changes your disability rating or eligibility for VA
+        benefits, you should expect this adjustment to be made in 1 to 2 months.</p>
+      );
+      break;
+    case STATUS_TYPES.withdrawn:
+      contents.title = 'You withdrew your appeal';
+      contents.description = (
+        <p>You have opted to not continue your appeal. If this information is incorrect, please
+        contact your Veteran Service Organization or representative for more information.</p>
+      );
+      break;
+    case STATUS_TYPES.ftr:
+      contents.title = 'Your appeal was closed';
+      contents.description = (
+        <p>You did not take an action VA requested in order to continue your appeal. If this
+        information is incorrect, or if you want to reopen your appeal, please contact your Veteran
+        Service Organization or representative for more information.</p>
+      );
+      break;
+    case STATUS_TYPES.ramp:
+      contents.title = 'You opted in to the Rapid Appeals Modernization Program (RAMP)';
+      contents.description = (
         <div>
-          For issues that are allowed, you will receive compensation. For more information, please
```

Comment:
Change 'For issues that are allowed, you will receive compensation. For more information, please contact your VSO or representative.' to 'If this decision changes your disability rating or your eligibility for VA benefits, you should expect this adjustment to be made in 1 to 2 months.'